### PR TITLE
Black/wallet items

### PR DESCRIPTION
**Pull request by MichaelEdwardBlack**

### CODE REVIEW

File: browser/react-wallet/package.json [hunk 1]
```diff
@@ -7,12 +7,17 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "@trinsic/trinsic": "^1.2.1",
```
Comment (fundthmcalculus) [inline browser/react-wallet/package.json:10]:
```suggestion
    "@trinsic/trinsic": "^1.3.0",
```

File: browser/react-wallet/scratch.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const trinsic = require('@trinsic/trinsic');
+
+let service = new trinsic.AccountService();
+let details = new trinsic.AccountDetails();
+details.setName("Michael Black");
+details.setEmail("michaelblack117@gmail.com");
```
Comment (fundthmcalculus) [inline browser/react-wallet/scratch.js:6]: you might want a bogus email here
```suggestion
details.setEmail("info@trinsic.id");
```

File: browser/react-wallet/src/actions/index.js [hunk 3]
```diff
@@ -1,12 +1,229 @@
...
+export const ERROR = 'ERROR';
+export const ResponseStatus = { 
+  "SUCCESS": 0,
+  "WALLET_ACCESS_DENIED": 10,
+  "WALLET_EXISTS": 11,
+  "UNAUTHENTICATED": 16,
+  "ITEM_NOT_FOUND": 20,
+  "SERIALIZATION_ERROR": 200,
+  "UNKNOWN_ERROR": 100
+}
```
Comment (fundthmcalculus) [inline browser/react-wallet/src/actions/index.js:17]: this should be a generated enum from the proto files.

File: browser/react-wallet/src/components/Inputs/DateInput.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+//     this.monthInput = React.createRef();
+//     this.yearInput = React.createRef();
+//   }
+
+//   resetState = () => {
+//     this.setState({
+//       day: "",
+//       month: "",
+//       year: "",
+//     })
```
Comment (fundthmcalculus) [inline browser/react-wallet/src/components/Inputs/DateInput.js:21]: All commented out code, maybe remove this?

File: dotnet/Blazor/Pages/FetchData.razor [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+            {
+                <tr>
+                    <td>@forecast.Date.ToShortDateString()</td>
+                    <td>@forecast.TemperatureC</td>
+                    <td>@forecast.TemperatureF</td>
+                    <td>@forecast.Summary</td>
+                </tr>
+            }
+        </tbody>
+    </table>*@
```
Comment (fundthmcalculus) [inline dotnet/Blazor/Pages/FetchData.razor:31]: commented out, or a template?

**Review by fundthmcalculus — APPROVED** [review]

File: dotnet/Blazor/Pages/FetchData.razor (hunk 5 above)
Comment (MichaelEdwardBlack) [reply dotnet/Blazor/Pages/FetchData.razor]: Template

File: browser/react-wallet/scratch.js (hunk 2 above)
Comment (MichaelEdwardBlack) [reply browser/react-wallet/scratch.js:6]: Oh shoot yeah I need to delete that file altogether as I was just using that to test some stuff early on

**Pull request merged**